### PR TITLE
[LM-9627] Added LMComment

### PR DESCRIPTION
**Pull request by prabha-likeminds**
## Description

**Features**
- Added LMComment component

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: src/components/LMComment/index.tsx [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+      setNumberOfLines(showText ? undefined : MAX_LINES);
+    }
+  }, [showText, showMoreButton]);
+
+  //creating content props as per customization
+  const updatedContentProps = commentContentProps
+    ? commentContentProps
+    : {
+        text: comment.text,
+        onTextLayout: (e: any) => onTextLayout(e),
```
Comment (ishaan1607) [inline src/components/LMComment/index.tsx]: why `e`? it should be named as `text` or `inputText`

File: src/constants/strings.ts [hunk 2]
```diff
@@ -8,4 +8,11 @@ export const LINK_ATTACHMENT_TYPE = 4;
...
-export const MAX_DEFAULT_POST_CONTENT_LINES = 3
\ No newline at end of file
+export const MAX_DEFAULT_POST_CONTENT_LINES = 3
+
+// default maximum number of lines to be visible in post comments
+export const MAX_DEFAULT_COMMENT_LINES = 3
+
+// comment levels
+export const PARENT_LEVEL_COMMENT = 0
+export const CHILD_LEVELCOMMENT = 1
```
Comment (ishaan1607) [inline src/constants/strings.ts]: rename `CHILD_LEVELCOMMENT` to `CHILD_LEVEL_COMMENT`

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

File: src/components/LMComment/index.tsx [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+import layout from '../../utils/layout';
+import {LMCommentProps} from './types';
+import LMButton from '../../base/LMButton';
+import LMInputText from '../../base/LMInputText';
+import {timeStamp} from '../../utils';
+import LMIcon from '../../base/LMIcon';
+import {PARENT_LEVEL_COMMENT} from '../../constants/strings';
+import {FlashList} from '@shopify/flash-list';
+
+const LMComment = ({
```
Comment (ishaan1607) [inline src/components/LMComment/index.tsx]: rename to `LMCommentItem`

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

File: src/components/LMComment/index.tsx (hunk 1 above)
Comment (prabha-likeminds) [reply src/components/LMComment/index.tsx]: It is basically an event that occurs whenever the layout of the text changes. This occur on load and return some native properties of the text like in how many lines the text will be rendered.

**Review by ishaan1607 — APPROVED** [review]

**Pull request merged**